src/app/components/toast/index.html

```diff
@@ -0,0 +1,92 @@
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="SkyToastConfig type">
+    <sky-demo-page-property
+      propertyName="message"
+      isOptional="true">
+      Sets the message that the toast displays. This may not be provided with the <stache-code>customComponentType</stache-code> property.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="customComponentType"
+      isOptional="true">
+      Sets the component that the toast renders. This may not be provided with the <stache-code>message</stache-code> property.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="providers"
+      isOptional="true">
+      Specifies an array of custom providers to pass to the <stache-code>customComponentType</stache-code>'s constructor.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="toastType"
+      isOptional="true">
+      Sets the type of the toast and determines the color and icon displayed. Available values are: <em>info, success, warning, and danger</em>.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="SkyToastMessage type">
+     <sky-demo-page-property
+      propertyName="close()">
+      Closes the linked toast and activates its <stache-code>closing</stache-code> and then <stache-code>closed</stache-code> states.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="isClosing">
+      An observable boolean showing if the toast is currently closing.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="isClosed">
+      An observable boolean showing if the toast is closed.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="message">
+      The message that the toast displays.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="customComponentType">
+      The component that the toast renders.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="providers">
+      An array of custom providers that were passed to the <stache-code>customComponentType</stache-code>'s constructor.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="toastType">
+      The type of the toast and determining the color and icon displayed. Available values are: <em>info, success, warning, and danger</em>.
```

Comment:
I thought the `toastType` property of `SkyToastConfig` specified this. ... "and determining ... displayed" doesn't work here. ", which determines ... to display" would work. ... Same question as before about why the values are italicized.

Reply:
I kinda just went with what we do for code tables in sky api docs for the italicized-ness

Reply:
I don't think we use that style anywhere in the SKY UX docs. I'd either format the 4 types as code snippets with the stache-code element or change "values" to "types" and leave the types unformatted. Either option works, but I'd opt for the code snippets.

Reply:
Also, would it make sense to change this first sentence to "The type of toast that was passed to the <stache-code>customComponentType</stache-code>'s constructor to determine the color and icon to display"?

Reply:
I switched to stache-codes

I wouldn't say that. It determines the styling and icon of the toast whether it has a customComponentType or not. I could change that sentence to "The type of toast that determines the color and icon displayed."

Reply:
That sounds good to me.